localisation/decisions.csv

```diff
@@ -367,8 +367,10 @@ invite_witch_king_of_angmar_desc;Since the Angmar War ended some one thousand ye
 ###########################;x }
 ##### ARNOR DECISIONS #####;x {
 ###########################;x
-accept_bre_hobbit_title;The hobbits of Breeland;x
-accept_bre_hobbit_desc;Breeland is inhabited by men and hobbit minority. By accepting that minority as part of our folk we can strenghten our country.;x
+shire_bree_unification_title;The unification of Breeland and Shire;x
+shire_bree_unification_desc;Now that we have asserted dominance over these land in the heartland of Eriador, we can unify them.;x
+the_buckland_question_title;The Buckland Question;x
+the_buckland_question_desc;Now that we have unified Bree and Shire, we should decide our policy regarding Buckland - an autonomous hobbit colony.;x
 accept_bck_hobbit_title;The hobbits of Buckland;x
```

Comment:
Could we get a consistent title capitalization? The U in `unification` and H in `hobbits` should be capitalized